.github/workflows/deploy.yml

```diff
@@ -27,17 +27,33 @@ jobs:
   deploy:
     name: Deploy
     runs-on: ubuntu-latest
+    container:
+        image: ubuntu:latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: ruby/setup-ruby@v1
+      - name: Update packages
+        run: |
+            apt-get update -qq
+      - name: Install dependencies
+        run: |
+            apt-get install -qq -y jq rsync git libyaml-0-2 npm
+      - name: Checkout git repository
+        uses: actions/checkout@v3
+      - name: Set GitHub workspace as git safe.directory # Required to work around: https://github.com/actions/checkout/issues/766
+        run: |
+            git config --global --add safe.directory "$GITHUB_WORKSPACE"
+      - name: Install Ruby
+        uses: ruby/setup-ruby@v1
         with:
-          bundler-cache: true
-      - uses: actions/setup-node@v2
+            bundler-cache: true
+        env:
+            ImageOS: ubuntu22
+      - name: Install Node.js
+        uses: actions/setup-node@v2
         with:
-          cache: "npm"
-      - name: Install dependencies
+            cache: "npm"
+      - name: Install Ruby dependencies
         run: |
-          bundle install
+            bundle install
```

Comment:
We can remove this because `bundler-cache: true` for `ruby/setup-ruby` does this.

Reply:
Good point! I misunderstood how the caching works. Thanks for pointing this out. I should have checked the `setup-ruby` documentation.

Reply:
Fixed.